[MI-3689]: Test cases of Zoom PR #284 and #287 (#9)
Brightscout/mattermost-test-management: data/test-cases/plugins/zoom/general/Message_edit.md

---
# (Required) Ensure all values are filled up

> name: "Change Test Title"

Reviewer: ```suggestion
name: "Change meeting title"
```

status: Active
priority: Normal
folder: Change Folder Name
authors: "@change_required"
team_ownership: 
- Change Team Name
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

1. Start a zoom meeting.

> 2. Edit the message presented to the user once the meeting is started.

Reviewer: ```suggestion
2. Edit the message presented to the users to join the meeting once the meeting is started.
```

steps_hashed: null
---

> **Step 1**Change Folder Name

Reviewer: ```suggestion
**Step 1**
```
